### ISSUE TITLE
Bug: Spell checker not working for multiple languages in slack recipe

### ISSUE THREAD

**Issue by cmeissner** [ferdium/ferdium-app#1321]
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/HEAD/CODE_OF_CONDUCT.md) that Ferdium Recipes adheres to.
- [X] I have searched the [issue tracker](https://github.com/ferdium/ferdium-app/issues?q=is%3Aopen+is%3Aissue+label%3A%22recipe+%3Abookmark_tabs%3A%22+label%3A%22bug+%3Abeetle%3A%22+) for a bug report that matches the one I want to file, without success.

### Ferdium Version

6.2.4

### Recipe Name

Slack

### Recipe Version

1.4.2

### Steps to reproduce

1. Set `Spell checking language` in `Language` on `Detect language automatically`
2. Try typing a text in a language other than English (tested with e.g. German & Italian) it won't work
3. Try typing a text in a language other than English in slack webui it would work

### Expected Behavior

Spellchecker should detect the correct language and would only show wrong words in the detected language.

### Actual Behavior

Spell checking won't work in languages other than English

### Screenshots

Example with German

![image](https://github.com/ferdium/ferdium-app/assets/14936920/795b9f33-abf5-4b56-9c00-92e7884e9e7d)

Example with Italian

![image](https://github.com/ferdium/ferdium-app/assets/14936920/a6d06879-e36d-4bbf-8bde-535bc04e2599)

### Additional Information

Auto-detection of language for spell checking seems to work in other receipts (e.g. gmail).

![image](https://github.com/ferdium/ferdium-app/assets/14936920/2b2a77f4-9793-44a1-9f8b-00ee7ac3ed0c)

**Comment by jvleminc** [comment]
Yep, having the same issue with auto-detect and Spanish, even setting the language to Spanish doesn't work. I am on 6.5.1 in Ubuntu 22.04.

![image](https://github.com/ferdium/ferdium-app/assets/45768550/58d9c86b-5556-4d5a-ba8f-943facc87276)

Version: 6.5.1
Server: Ferdium Server
Electron: 25.8.4
Chrome: 114.0.5735.289
Node.js: 18.15.0
Platform: Linux 6.2
Arch: x64
Build date: Mon Oct 02 2023 23:12:48 GMT-0300 (Chile Summer Time)
Git SHA: fbe67ea3
Git branch: release

![image](https://github.com/ferdium/ferdium-app/assets/45768550/29f6c896-8c5c-4c34-9c75-47dda3ac1b7a)

It's also happening in (at least) whatsapp and messenger
![image](https://github.com/ferdium/ferdium-app/assets/45768550/af23b751-95c9-4225-87af-5289b0c4137a)
![image](https://github.com/ferdium/ferdium-app/assets/45768550/d10b5311-3d03-43a5-987a-005d268123f4)

**Comment by jvleminc** [comment]
Still present in 6.6.0

**Comment by AliMehraei** [comment]
Same issue with English and Persian languages...

**Comment by unstressable** [comment]
In 6.7.0 the spell checker does not work for german.
It always suggests english words.

Example: GMail Recipe:
![image](https://github.com/ferdium/ferdium-app/assets/50423935/55e8200c-2886-412c-bb96-cba1323dfc9c)

**Comment by Mat-DB** [comment]
Same issue.
When setting it to `Detect language automatically` it does not work but when setting the language to Dutch it does work.
Only problem I use Dutch and English.

Just an idea no clue how difficult this is, add support to add multiple languages (just like in Firefox).
Version 6.7.2
Edit: after making this comment it started working sometimes. When a sentence is long enough to detect it, what makes sense.

**Comment by klonos** [comment]
QQ (and sorry if not directly related to the issue reported here): where is the dictionary being kept for Slack within Ferdium? ...I have accidentally added a word that I want to remove, and it cannot be found in `~/Library/Spelling/LocalDictionary` (which is the global dictionary in MacOS for other apps)

**Comment by jvleminc** [comment]
6.7.6 on Linux 22.04: still not working.

**Comment by SimeonEhrig** [comment]
Can confirm. Same problem with Mattermost and German and English.

> Just an idea no clue how difficult this is, add support to add multiple languages (just like in Firefox).

I would also prefer this option. Works perfectly in Firefox.

**Comment by marcolopes** [comment]
Spell Checker only works with EN...
Tried many workarounds and nothing!
It does not download any languages (except EN) to AppData\Roaming*Ferdium\Dictionaries*
Even tried to overlap PT language from Chrome over Ferdium EN with no success!
(Windows 10)
